cordova-plugin-outline/apple/src/OutlinePlugin.swift

```diff
@@ -58,6 +58,10 @@ class OutlinePlugin: CDVPlugin {
           name: NSNotification.Name(rawValue: OutlinePlugin.kAppQuitNotification),
           object: nil)
     #endif
+
+    #if os(iOS)
```

Comment:
I'm not yet sure whether it does. If it applies when I update `cordova-osx`, I'll remove the ifdef on the `alalama-cordova-update` branch.

Reply:
Oh, I see. You are not upgrading the macOS client yet. OK. Can you add a TODO then?

Reply:
Since these changes are not going to master yet and are scoped for iOS only, I don't think it's necessary to add the TODO. The app builds and runs as before in macOS.